### PR TITLE
Assignment/assignment7

### PR DESCRIPTION
**Pull request by SarahYacoub6**
kindly review the assignment,  provide me if any comment.

### CODE REVIEW

File: assignment/assignment7/stylesheet.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
+@import 'reset';
+$gray_color:dimgray;
+@mixin set_width($value)
+{
+    width: $value;
+}
+.float_left
```
Comment (AhmadBaracat) [inline assignment/assignment7/stylesheet.scss]: Are you sure you need a class for just floating left?

File: assignment/assignment7/stylesheet.scss [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    background-color: white;
+    @include set_width(60%);
+    margin: auto;
+    margin-top: 20px;
+    overflow: hidden;
+}
+header
+{
+    padding: 10px;
+	border-bottom:2px solid #000;
```
Comment (AhmadBaracat) [inline assignment/assignment7/stylesheet.scss:24]: Indentation :)

**Review by AhmadBaracat — APPROVED** [review]
Just look at my comments.

File: assignment/assignment7/stylesheet.scss (hunk 1 above)
Comment (SarahYacoub6) [reply assignment/assignment7/stylesheet.scss]: I know I don't need to create class to just foat :right but I used it to access all sass features

**Pull request merged**